app/views/draft_mailer/draft_published_notification.html.erb

```diff
@@ -9,9 +9,16 @@
   <p>
     <%= @user[:name] %>,
   </p>
+<% env = Rails.configuration.cmr_env.upcase %>
+<% if env != 'OPS'%>
+  <p>
+    <%= "Your collection metadata record #{@short_name}_#{@version} has been successfully #{ @is_update ? 'updated' : "published to the CMR (#{env})"}. Your collection's concept ID is #{@concept_id}." %>
```

Comment:
same comments about `cmr_env` vs the `Rails.env`
also, i think you can do this without the if/else blocks for the content. I would do it like:
```
<% add_env_if_needed = Rails.env.production? nil : " (#{Rails.env})" %>
<p>
  <%= "same string message as before but add #{add_env_if_needed} rest of the message" %>
</p>
```

Reply:
I think this change can be applied to many of the files below.

Reply:
I see, because the content is mostly the same, only somewhat different?

Reply:
yes and if it's `nil` it won't add anything